FINAL.sh

```diff
@@ -0,0 +1,28 @@
+# Part 1: Takes unique gender-yearsExperience combos in wages.csv, sorts
```

Comment:
1.	1
2.	1
3.	Almost, your code works but doesn’t give quite the right answer. Your call to grep doesn’t pull all the 16 or 12 years of school records. Something like the following would work: cat wages.csv | cut -d , -f 3,4 | grep '16,' |  sort -t , -k 2  -n| head -n 1 | cut -d , -f 2; cat wages.csv | cut -d , -f 3,4 | grep '12,' |  sort -t , -k 2 -n | head -n 1 | cut -d , -f 2
a.	0.7

2.7/3